### PR TITLE
Spark-nlp in 30 seconds

### PR DESCRIPTION
**Pull request by saifjsl**
## Description

Includes changes that make Spark-NLP easy for new comers, by calling `sparknlp.session()`. Also changes PretrainedPipeline api in order to easy retrieve pretrained pipelines.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: python/example/model-downloader/ModelDownloaderExample.ipynb [hunk 1]
```diff
@@ -198,9 +198,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from sparknlp.pretrained.pipeline.en import SentimentPipeline\n",
-    "\n",
-    "SentimentPipeline.annotate(\"This is a good movie!!!\")"
+    "PretrainedPipeline(\"pipeline_vivekn\").annotate(\"This is a good movie!!!\")"
```
Comment (maziyarpanahi) [inline python/example/model-downloader/ModelDownloaderExample.ipynb]: Shouldn't this be `sentiment_vivken`? We know already it is a pipeline due to `PretrainedPipeline`, I think `vivekn` on its own is not clear, so we can combine the annotator with its algorithm/method: `sentiment_vivekn`.

File: python/sparknlp/pretrained.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from sparknlp.annotator import *
+from pyspark.ml.wrapper import JavaModel
+from pyspark.sql import DataFrame
+from sparknlp.base import LightPipeline
+
+
+class ResourceDownloader(object):
+
+    @staticmethod
+    def downloadModel(reader, name, language, remote_loc=None):
```
Comment (maziyarpanahi) [inline python/sparknlp/pretrained.py]: Can we also add another field called `source` which represents the source of the dataset which we built the model? For instance, we will have multiple Lemma in Italian one from DXC, one from UD, one from the famous GitHub repo for Lemma Dictionaries, etc.
This way, we can have `source` default to one of these models, and the user can choose whether they wan POS Tagger in English based on regular format or the one from the UD treebank.

File: src/main/scala/com/johnsnowlabs/nlp/pretrained/PretrainedPipeline.scala [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package com.johnsnowlabs.nlp.pretrained
+
+import com.johnsnowlabs.nlp.LightPipeline
+import org.apache.spark.ml.PipelineModel
+import org.apache.spark.sql.DataFrame
+
+case class PretrainedPipeline(
+                               downloadName: String,
+                               language: String = "en",
+                               source: String = ResourceDownloader.publicLoc
```
Comment (maziyarpanahi) [inline src/main/scala/com/johnsnowlabs/nlp/pretrained/PretrainedPipeline.scala]: Shoot! I can see we have a filed here called `source` which is about the location of the model. If we decide to add a field to allow users to choose which model they want for one specific annotator, maybe we can choose `type` or something or rename this to `url` and use the `source`.

File: src/test/scala/com/johnsnowlabs/nlp/annotators/ner/dl/NerDLReader.scala [hunk 4]
```diff
@@ -28,13 +28,13 @@ class NerDLReaderTestSpec extends FlatSpec {
     val spark = ResourceHelper.spark
     import spark.implicits._
 
-    val bp = BasicPipeline().pretrained()
+    val bp = PretrainedPipeline("pipeline_basic")
```
Comment (maziyarpanahi) [inline src/test/scala/com/johnsnowlabs/nlp/annotators/ner/dl/NerDLReader.scala]: Should we just call this `basic` instead of `pipeline_basic`?

Comment (saifjsl) [reply src/test/scala/com/johnsnowlabs/nlp/annotators/ner/dl/NerDLReader.scala]: subject to renew, we should create a new list of pipelines we want to have. this specific test is using that one so we might keep it just fbecause

File: src/main/scala/com/johnsnowlabs/nlp/pretrained/PretrainedPipeline.scala (hunk 3 above)
Comment (saifjsl) [reply src/main/scala/com/johnsnowlabs/nlp/pretrained/PretrainedPipeline.scala]: right, agreed. I'll rename that parameter

File: python/example/model-downloader/ModelDownloaderExample.ipynb (hunk 1 above)
Comment (saifjsl) [reply python/example/model-downloader/ModelDownloaderExample.ipynb]: disregard model and pipeline names for now :) they should be all remade from scratch

File: python/sparknlp/pretrained.py (hunk 2 above)
Comment (saifjsl) [reply python/sparknlp/pretrained.py]: For now we could name it just differently, once we get to a point to multiple sources per public model we can re-consider

**Pull request merged**